scenarioo-server/src/main/java/org/scenarioo/business/diffViewer/comparator/ComparatorParameter.java

```diff
@@ -0,0 +1,38 @@
+package org.scenarioo.business.diffViewer.comparator;
+
+import org.scenarioo.model.configuration.ComparisonConfiguration;
+
+import java.awt.*;
+
+public class ComparatorParameter {
```

Comment:
I think this should be called `ComparisonParameters` cause it contains all parameters for one comparison.